atomic_reactor/plugins/input_osv3.py

```diff
@@ -202,7 +211,12 @@ def run(self):
             'source': {
                 'provider': 'git',
                 'uri': git_url,
-                'provider_params': {'git_commit': git_ref}
+                'provider_params': {
+                    'git_commit': git_ref,
+                    'container_yaml': container_yaml,
+                    'git_commit_depth': git_commit_depth,
+                    'git_branch': git_branch
+                }
```

Comment:
Minor nit: often nicer on these last two lines to end with a comma even though it isn't strictly needed, as it can make future diffs in this area easier to read.